src/lib/db.js

```diff
@@ -169,12 +169,12 @@ export function articleIssueQuery(slugs) {
       .select('issue_order as issueNumber', 'articles.slug as slug')
       .from('articles')
       .innerJoin(
-        'issues_posts_order',
-        'issues_posts_order.article_id',
+        'issues_articles_order',
```

Comment:
Same as above -  not going to mark all of these here on out since it's a Find&Replace and there are tons of instances 😆 .